sample/src/main/java/com/thoughtworks/ark/sample/sound/SoundViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.thoughtworks.ark.sample.sound
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import com.thoughtworks.ark.sound.alert.AlertItem
+import com.thoughtworks.ark.sound.alert.AlertPlayerImpl
+import com.thoughtworks.ark.sound.media.MediaItem
+import com.thoughtworks.ark.sound.media.MediaPlayerImpl
+import com.thoughtworks.ark.sound.tts.TTSItem
+import com.thoughtworks.ark.sound.tts.TTSSpeakerImpl
+
+class SoundViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
Inject the players such as different dispatchers: IoDispatcher, UIDispatcher ... would be better.